rustls/src/client/tls12.rs

```diff
@@ -974,6 +977,12 @@ impl ExpectFinished {
             return;
         }
 
+        let now = if let Ok(now) = self.config.get_current_time() {
+            now
+        } else {
+            return;
```

Comment:
Does this early return merit a `debug!` like on L976? I think it would be helpful.